### ISSUE TITLE
I am not able to find source code location for com.couchbase.client

### ISSUE THREAD

**Issue by tejo93** [couchbase/couchbase-jvm-clients#30]
Hi Team ,

 I am trying to find source code location for 

https://mvnrepository.com/artifact/com.couchbase.client/tracing-opentelemetry/0.3.3

I could not find it in Couchbase repository. Please help here!

**Comment by programmatix** [comment]
Hi @tejo93 please see https://github.com/couchbase/couchbase-jvm-clients/tree/master/tracing-opentelemetry

**Comment by tejo93** [comment]
Hi @programmatix  , can you please let me know the source code location for the version / tag : 0.3.3 
I followed the link which you shared , but didn't get the version I am looking for.

**Comment by programmatix** [comment]
It was updated in this commit https://github.com/couchbase/couchbase-jvm-clients/commit/5c4a9945b82f026be2ed10e08ecfeab6ef2d2543 so https://github.com/couchbase/couchbase-jvm-clients/tree/5c4a9945b82f026be2ed10e08ecfeab6ef2d2543/tracing-opentelemetry will give you the source at that point in history.

We do tag the individual modules now, but this is a very old commit, and before we started that practice.

Hope this helps.
